recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
     - utf_8_mac.patch  # [osx]
 
 build:
-  number: 0
```

Comment:
I'm a bit lost in the shuffle, but please check if bn 0 was marked as broken.

Reply:
It's indeed marked as broken on Windows - https://github.com/conda-forge/admin-requests/pull/1395.